layout/hud/tab-menu.xml

```diff
@@ -51,6 +51,10 @@
 				<Frame src="file://{resources}/layout/pages/end-of-run/end-of-run.xml" hittest="true" hittestchildren="true" />
 			</Panel>
 
+			<Panel id="ZoningContainer" class="hud-tab-menu__zoning" hittestchildren="true">
```

Comment:
Isn't `hittestchildren` redundant here, since it's immediate child has `hittest` and `hittestchildren`? Don't know exactly how this works in code.

Reply:
could be changed, the user can't interact with this in any way currently. I copied this line from my zoning UI branch.